docs/conf/portland/2019/unconference.rst

```diff
@@ -6,11 +6,19 @@ Unconference
 
 {% include "conf/events/unconference.rst" %}
 
+Unconference sessions are held downstairs from the Crystal Ballroom where the main stage talks are held, in Lola's Room. Because the unconference has become so popular in recent years, we ask session organizers to bear the following guidelines in mind:
```

Comment:
```suggestion
Unconference sessions are held downstairs from the Crystal Ballroom, in Lola's Room. Because the unconference has become so popular in recent years, we ask session organizers to bear the following guidelines in mind:
```

possibly less potential for confusion?